src/wallet/wallet.h

```diff
@@ -604,17 +612,28 @@ class COutput
     }
 };
 
+/** Parameters for one iteration of Coin Selection. */
 struct CoinSelectionParams
 {
+    /** When true, try to use the Branch and Bound algorithm before falling back to knapsack. */
     bool use_bnb = true;
+    /** Size of a change output in bytes, determined by the output type. */
     size_t change_output_size = 0;
+    /** Size of the input to spend a change output in virtual bytes. */
     size_t change_spend_size = 0;
+    /** The targeted feerate of the transaction being built. */
     CFeeRate m_effective_feerate;
+    /** The feerate estimate used to calculate the upper bound of the cost to spend a change output. */
```

Comment:
Mh, this one is a bit more subtle. How about:
```suggestion
    /** The feerate used to estimate an upper bound on what should be sufficient to spend the change output some time in the future. */